docs/DafnyRef/integration-py/IntegrationPython.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Integrating Dafny and Python code
+---
+
+The Dafny compilation process translates Dafny programs into target language
+source code and then potentially runs the result. 
+
+The Dafny-to-Python compiler writes out the translated files of a file _A_`.dfy`
+to a folder _A_`-py`. The `-out` option can be used to choose a
+different output folder. The input `.dfy` files are translated to various `.py` files 
+which are placed in the output folder along with helper files.
+If more than one `.dfy` file is listed on the command-line, then the output
+folder name is taken from the first file.
+
+A multi-language program that combines Dafny and Python
+code "just" needs to be sure that the translated Dafny code fits in
+to the Python code. There are two aspects to this:
+- ensuring that the names of entities in the translated Dafny code are usable from Python
+- ensuring that the types are the same on both sides
+
+## **The Dafny runtime library**
+
+The step of compiling or running Python files requires the Dafny runtime library for Python. That library is automatically included if dafny is doing the compilation,
+but not if dafny is only doing translation.
+
+## **Manually executing Dafny-generated Python code**
+
+Suppose a Dafny program is contained in a `.dfy` file, `A.dfy`, which also contains the `Main` method. One can build the corresponding Java program (without running it) using this command:
+
+`dafny build --target:py A.dfy`
```

Comment:
Can you choose between `-t: and `--target:` for the documentation? I would prefer to put `--target:` as this is easier to remember than `-t`, but I'm fine with both. Just want to be consistent.

Reply:
Made them all --target